cli/src/semgrep/core_runner.py

```diff
@@ -456,6 +457,7 @@ def execute(self) -> int:
 
         terminal = get_state().terminal
         with Progress(
+            TextColumn(" "),
```

Comment:
this indents by two spaces (one space of text, one space of column gap) lining the progress bar up with the rest of semgrep's output